subprojects/ftxui.wrap

```diff
@@ -0,0 +1,11 @@
+[wrap-file]
+directory = FTXUI-2.0.0
+source_url = https://github.com/ArthurSonzogni/FTXUI/archive/refs/tags/v2.0.0.tar.gz
+source_filename = FTXUI-2.0.0.tar.gz
+source_hash = d891695ef22176f0c09f8261a37af9ad5b262dd670a81e6b83661a23abc2c54f
+patch_directory = ftxui
+
+[provide]
+ftxui-screen = screen_dep
+ftxui-dom = dom_dep
+ftxui-component = component_dep
```

Comment:
Upstream installs *one* dependency, `fxtui`, and would require finding via meson via `dependency('fxtui', method: 'cmake', modules: [fxtui::screen'])`.

How utterly impossible to map to pkg-config at all.

I guess this is the best we could do.